tracing/lightstep.go

```diff
@@ -0,0 +1,52 @@
+package tracing
+
+import (
+	"go.opentelemetry.io/collector/translator/conventions"
+	"go.opentelemetry.io/otel/exporters/otlp"
+	"go.opentelemetry.io/otel/label"
+	export "go.opentelemetry.io/otel/sdk/export/trace"
+	"go.opentelemetry.io/otel/sdk/resource"
+	"google.golang.org/grpc/credentials"
+)
+
+// Lightstep service to export traces to
+type Lightstep struct {
+	Endpoint string `long:"lightstep-endpoint" description:"lightstep endpoint" default:"ingest.lightstep.com:443"`
+	Token    string `long:"lightstep-token" description:"lightstep access token"`
+	Service  string `long:"lightstep-service"  description:"lightstep process service name" default:"concourse-web"`
+}
+
+// IsConfigured identifies if an endpoint has been set
+func (s Lightstep) IsConfigured() bool {
+	return s.Token != ""
+}
+
+// Exporter returns a SpanExporter to sync spans to Lightstep
+func (s Lightstep) Exporter() (export.SpanSyncer, error) {
+	headers := map[string]string{
+		"lightstep-access-token": s.Token,
+	}
+
+	secureOption := otlp.WithTLSCredentials(credentials.NewClientTLSFromCert(nil, ""))
+
+	exporter, err := otlp.NewExporter(
+		secureOption,
+		otlp.WithAddress(s.Endpoint),
+		otlp.WithHeaders(headers),
```

Comment:
...instead we could just pass the headers we parsed from the command line as `s.Headers`.